airbyte-integrations/connectors/source-microsoft-teams/integration_tests/configured_catalog.json

```diff
@@ -0,0 +1,112 @@
+{
```

Comment:
if all stream are here we don't this file, SAT will take discovery output as configured_catalog